Source/Core/DiscIO/NANDImporter.cpp

```diff
@@ -108,12 +119,19 @@ std::string NANDImporter::GetPath(const NANDFSTEntry& entry, const std::string&
   return parent_path + '/' + name;
 }
 
+std::string NANDImporter::FormatDebugString(const NANDFSTEntry& entry)
+{
+  return StringFromFormat(
+      "%12.12s 0x%02hhx 0x%02hhx 0x%04hx 0x%04hx 0x%08x 0x%04hx 0x%04hx 0x%04hx 0x%08x", entry.name,
```

Comment:
I'm pretty sure Windows doesn't support ```hh``` length prefix here, https://msdn.microsoft.com/en-us/library/56e442dc.aspx

They should get promoted to int anyway.